Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8-slim
```

Comment:
how does slim compare with alpine? Is there any cons?

Reply:
I am happy that you caught it, no cons, I was just confused what to pick, do you think it's better to pick alpine? also do you think it't better to pick latest LTS instead of freezing to 8?

Reply:
Alpine will be lighter. I haven't tried slim yet but all my previous project have no problem with alpine. Also think we should go with latest LTS